config/locales/de.yml

```diff
@@ -3212,6 +3212,40 @@ de:
         - fahre Dein Gerät herunter, starte es erneut und versuch es nochmal.<br>
         - lade die Dateien einzeln hoch.<br>
         Bei wiederholtem Auftreten kontaktiere bitte das MaMpf-Team.
+  registration:
```

Comment:
Can you put the locales in the locales/registration/de.yml file (same for english)? These general locale files have become already way too large.